container-storage-setup.sh

```diff
@@ -755,26 +756,33 @@ create_partition() {
     Fatal "udevadm settle after partition creation failed. Exiting."
   fi
 
-  if ! wait_for_dev ${dev}1; then
-    Fatal "Partition device ${dev}1 is not available"
+  part=$(query_first_child $dev)
+
+  if ! wait_for_dev ${part}; then
+    Fatal "Partition device ${part} is not available"
   fi
+  echo $part
+}
+
+query_first_child() {
+  lsblk -npl -o NAME "$1" | tail -n +2 | head -1
```

Comment:
I was thinking that it is easier to understand. If we are querying first child, then take first two lines of output. That will have parent and first child. 

tail -n +2, will return you all children. So I think it is simpler to do just first two lines?